OneCommunityGlobal/HighestGoodNetworkApp
# Zijie lint fix for reports component Phase 3

# Description
Lint fix for Reports component, this is phase 3.

## Files changed
WbsTable directory

- WbsTable/WbsTable.jsx 
- WbsTable/index.js

# How to Test

To test this pull request:

- Check out the current branch
- Run `npm install` and `npm run start:local` to run this PR locally  
- Log in as an admin user
- Go to Reports,  Check functionality is the same as before. (Please record a video)
- Check the functionality of the report WbsTable is normal. (Please record a video)
- Check all the functionality of the report is the same as before, and see if any bugs have appeared. (Please record a video)

Optional:
- Set up ESLint in VSCode/ IntelliJ IDEA, I personally use IntelliJ IDEA
- Delete .eslintignore file in the root: `rm .eslintignore`
- `npx eslint src/components/Reports/WbsTable/WbsTable.jsx`
- `npx eslint src/components/Reports/WbsTable/index.js`
- `git restore .`

I have complete confidence that **the code will pass the ESLint tests.** Therefore, I kindly request that you **focus on testing the frontend**. Specifically, please ensure that the **report functionality remains consistent** with how it operated previously. Thank you for your time and effort!

Screenshot:
<img width="1263" alt="CleanShot 2023-09-30 at 13 11 18@2x" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46791134/472c249e-85dc-4222-93c5-9fd39fc5c71d">
